jobserver/templates/partials/header.html

```diff
@@ -17,7 +17,7 @@
       </ul>
 
       {% if not user.is_authenticated %}
-        <a class="btn btn-outline-light mt-2 mb-3 my-lg-0" href="{% url "social:begin" "github" %}?next={{ request.path }}" rel="nofollow">Login</a>
```

Comment:
There's another login link somewhere else. I can't remember exactly where but if you search `social:begin` you'll find it. I know this because I missed it out when I added the `rel="nofollow"` attribute and George had to point it out to me :smile:

Reply:
`applications/templates/applications/sign_in.html` 🙂